src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/service/ExternalUsersApiService.kt

```diff
@@ -94,18 +89,16 @@ class ExternalUsersApiService(
       .bodyValue(roleAmendment)
       .retrieve()
       .toBodilessEntity()
-      .block(timeout)
+      .block()
   }
-
-  @Throws(RoleNotFoundException::class)
   fun updateRoleAdminType(roleCode: String, roleAmendment: RoleAdminTypeAmendment) {
     log.debug("Updating role for {} with {}", roleCode, roleAmendment)
     externalUsersWebClient.put()
       .uri("/roles/$roleCode/admintype")
       .bodyValue(mapOf("adminType" to roleAmendment.adminType.addDpsAdmTypeIfRequiredAsList()))
       .retrieve()
       .toBodilessEntity()
-      .block(timeout)
+      .block() ?: throw RuntimeException("Failed to update role")
```

Comment:
I'm still unsure about throwing a runtimeexception.

Reply:
removed and tough we may never get into this scenario but changed to fail as null pointer and to through 500 (internal server error)